x/observer/genesis.go

```diff
@@ -11,8 +11,10 @@ import (
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	genesisObservers := genState.Observers
 	types.VerifyObserverMapper(genesisObservers)
+	observerCount := uint64(0)
 	for _, mapper := range genesisObservers {
 		k.SetObserverMapper(ctx, mapper)
+		observerCount += uint64(len(mapper.ObserverList))
```

Comment:
```suggestion
		observerCount += len(mapper.ObserverList)
```
Not blocking but I don't think the cast is necessary